sunspot/lib/sunspot/query/sort_composite.rb

```diff
@@ -19,12 +19,29 @@ def <<(sort)
       end
 
       # 
-      # Combine the sorts into a single param by joining them
+      # Combine the sorts into a single sort-param by joining them and add
+      # possible custom additional params
       #
       def to_params(prefix = "")
         unless @sorts.empty?
           key = "#{prefix}sort".to_sym
-          { key => @sorts.map { |sort| sort.to_param } * ', ' }
+          params = { key => @sorts.map { |sort| sort.to_params[:sort] } * ', ' }
```

Comment:
Since this method is pretty chunky, we should be looking to pull stuff out where we can - this separate `params` variable seems like a good candidate.